app/presenters/tree_node/hash.rb

```diff
@@ -10,7 +10,8 @@ class Hash < Node
 
     set_attribute(:hide_checkbox) { @object.key?(:hideCheckbox) && @object[:hideCheckbox] ? true : nil }
 
-    set_attribute(:selected) { @object.key?(:select) && @object[:select] ? true : nil }
+    # set_attribute(:selected) { @object.key?(:select) && @object[:select] ? true : nil }
+    set_attribute(:selected) { @object[:select] if @object.key?(:select) }
```

Comment:
This definitely needs to be tested against other trees using hash nodes & checkboxes ...